Houseclub/src/main/java/me/grishka/houseclub/fragments/ProfileFragment.java

```diff
@@ -295,4 +319,40 @@ public void onError(ErrorResponse error){
 				.setNegativeButton(R.string.cancel, null)
 				.show();
 	}
+
+	@Override
+	public void onActivityResult(int requestCode, int resultCode, Intent data)
+	{
+		if (requestCode == ProfileFragment.IMAGE_PICKER) {
+			if (data == null) {
```

Comment:
You're supposed to check `resultCode == Activity.RESULT_OK`, in which case the intent won't be null.